web/web.go

```diff
@@ -146,9 +146,13 @@ func New(st local.Storage, qe *promql.Engine, rm *rules.Manager, status *Prometh
 	instrf := prometheus.InstrumentHandlerFunc
 	instrh := prometheus.InstrumentHandler
 
-	router.Get("/", instrf("status", h.status))
+	router.Get("/", instrf("graph", h.graph))
```

Comment:
I prefer the suggestion of adding a input box to the existing status page for graphs. 

As developers it's likely our usage is atypical, and that we use /graph far more than the average user. Personally I hit the console templates more often than /graph, and /status frequently too.